homeassistant/components/entur_public_transport/sensor.py

```diff
@@ -55,6 +59,8 @@
         vol.Optional(CONF_NUMBER_OF_DEPARTURES, default=2): vol.All(
             cv.positive_int, vol.Range(min=2, max=10)
         ),
+        # Default doesn't really matter here, we'll always overwrite it anyway:
```

Comment:
YAML config is no longer to be expanded.

Reply:
Thanks @balloob . Do you have a pointer into forums/Discord for me to check how to go about it?